exercises/APE7/0702.tex

```diff
@@ -0,0 +1,53 @@
+\paragraph{a.} Let's first define what a belief vector is : for $\pi \in  \times_{s \in S} \Delta Y_s $, $i \in N$, $s \in S_i$, and $x \in Y_s$, $\pi_s(x)$ is the probability assigned by player $i$ to be at node $x$ given the fact that $i$ knows his information state $s$. ($X$ the nodes, $Y_s\in X$ is the set of nodes that have information $s$). \\
+We can calculate it we Baye's formula $$ \pi_s(y) = \frac{P(y | \tau, x^0)}{\sum_{x \in Y_s} P(x | \tau, x^0)}. $$
+
+Here there is four information states: 1.1, 2.2 via $\text{a}_1$, 2.2 via $\text{b}_1$ and 2.3: 
+
+\begin{itemize}
+\item $\pi_{1.1}= 1$ 
+\item $\pi_{2.2.a_1} = \frac{0.6}{0.6+0.4 \cdot 0.25} = \frac{6}{7}$
+\item $\pi_{2.2.b_1} = \frac{0.4\cdot 0.25}{0.6+0.4 \cdot 0.25} = \frac{1}{7}$ 
+\item $\pi_{2.3}     = 1 $
+\end{itemize}
+$\pi_{1.1}=\pi_{2.3} = 1$ because there is only one possible way to be at 1.1 (or 2.3) knowing that the information state is 1.1 (or 2.3).
+
+\paragraph{b.} The sequential value of player $i$ for the move $d$ and information state $s$ $$ U_i(d| s, (\tau, \pi) ) = \sum_{x \in Y_s} \pi_s(x) U_i((\tau_{-i}, d), x)$$
+Meaning that we will only care about nodes x with information $s$. \\
+We obtain : 
+\begin{itemize}
+\item $U_{1.1} (\text{a}_1) = \pi_{1.1} (0.9 \cdot 1 + 0.1 \cdot 8) = 2.6$
```

Comment:
Write it `U_1(a_1|1.1)` to be consistent with Definition 4.22 of the course notes